support managed identity for "azurerm_spring_cloud_app" (#8336)
terraform-providers/terraform-provider-azurerm: website/docs/r/spring_cloud_app.html.markdown

* `resource_group_name` - (Required) Specifies the name of the resource group in which to create the Spring Cloud Application. Changing this forces a new resource to be created.

* `service_name` - (Required) Specifies the name of the Spring Cloud Service resource. Changing this forces a new resource to be created.

> * `identity` - (Optional) A Managed Service Identity block as defined below.

Reviewer: ```suggestion
* `identity` - (Optional) An `identity` block as defined below.
```

---

A `identity` block supports the following:

* `type` - (Required) Specifies the identity type of the Spring Cloud Application. Possible value is `SystemAssigned`.

## Attributes Reference

The following attributes are exported:

* `id` - The ID of the Spring Cloud Application.

> * `identity` - An `identity` block as defined below, which contains the Managed Service Identity information for this Spring Cloud Application.

Reviewer: this is an argument so it's already exported

```suggestion
```
